pom.xml

```diff
@@ -227,6 +227,11 @@
                     <mavenExecutorId>forked-path</mavenExecutorId>
                 </configuration>
             </plugin>
+			<plugin>
```

Comment:
Pinning dependency versions is always a good idea 😄 
And including it here means that `dependency:go-offline` will download it.